crates/ef-testing/src/evm_sequencer/sequencer/v0.rs

```diff
@@ -23,12 +23,11 @@ lazy_static! {
 
         let storage = [
             ("Ownable_owner", *KAKAROT_OWNER_ADDRESS.0.key()),
-            ("native_token_address", *ETH_FEE_TOKEN_ADDRESS.0.key()),
-            ("contract_account_class_hash", CONTRACT_ACCOUNT_CLASS_HASH.0),
-            ("externally_owned_account_class_hash", EOA_CLASS_HASH.0),
-            ("account_proxy_class_hash", PROXY_CLASS_HASH.0),
-            ("precompiles_class_hash", PRECOMPILES_CLASS_HASH.0),
-            ("block_gas_limit", StarkFelt::from(BLOCK_GAS_LIMIT))
+            ("Kakarot_native_token_address", *ETH_FEE_TOKEN_ADDRESS.0.key()),
```

Comment:
Should as these be moved to a constants file?

Reply:
I think it can stay there but just use the constants